### DISCUSSION TITLE
Cannot use iframe

### DISCUSSION THREAD

**Discussion by waclawlabuda** [requarks/wiki#3399] [Help / Questions]
**Question**
I cannot insert iframe in my wiki page, any kind, even simple iframe example. It is not rendered in page view. I can see this as a plain text, not interpreted as html iframe tag.
![image](https://user-images.githubusercontent.com/35487956/86561647-6d9d9c00-bf61-11ea-84db-063ce7594133.png)

**Host Info (please complete the following information):**
OS: Docker
Wiki.js version: 2.4
Database engine: postgres 11

**Comment by Smankusors** [comment]
Duplicate of #1968

**Comment by waclawlabuda** [comment]
Thanks for your reply, it works. Is it possible to use iframe also in markdown? Now I can only use it in HTML page
